### DISCUSSION TITLE
Ypsomed Diasend: import carbs

### DISCUSSION THREAD

**Discussion by GreenRover** [NightscoutFoundation/xDrip#2694] [Q&A]
For now i use xdrip+ in companion mode with a nighscout what works excellent.
What i miss is getting the carbs. Those are not possible to import via companion mode.
Having carbs to suppress alerts after eating carbs /hypo correction) would help.

Are there any solutions?

**Comment by Navid200** [comment]
I don't believe any of xDrip alerts is affected by carbs.

**Comment by feeblee** [comment]
Assuming you are using myLife CamAPS FX with Ypsopump, due to the topic of your post.

I use this system and gather the CGM data with xDrip+ (Companion App Collector) to upload it to nightscout and display it on Apple Watch. To my best knowledge this doesn't work for Carb and Insulin data.
As Carb & Insulin data from CamAPS FX is uploaded to diasend (soon to glooko), I use the Diasend Bridge on nightscout to get this data. Only caveat is the once-every-hour upload restriction on diasend-side, that leads to delayed syncs. Might be unsatisfactory, if postprandial alerts arise before the diasend-sync, but might prevent some of them. 
I don't know, if this connection will be available with glooko in the future.
